autogen/agentchat/conversable_agent.py

```diff
@@ -131,7 +135,11 @@ def __init__(
 
         self._name = name
         # a dictionary of conversations, default value is list
-        self._oai_messages = defaultdict(list)
+        if messages is None:
+            self._oai_messages = defaultdict(list)
+        else:
+            self._oai_messages = messages
+
         self._oai_system_message = [{"content": system_message, "role": "system"}]
```

Comment:
Just a comment not for this PR: we should also allow a list of messages for the `system_message`, probably creating a new argument for it `system_messages`. Different from `chat_messages`, `system_messages` are not removed after `initiate_chat` by default.